boards/shields/ssd1306_128x64/boards/frdm_k64f.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_I2C=y
```

Comment:
Redundant with same in ` boards/shields/ssd1306_128x64/ssd1306_128x64.conf`

Reply:
thanks, fixed